translator/internaldata/traces_to_oc.go

```diff
@@ -332,7 +331,7 @@ func linksToOC(links data.SpanLinkSlice, droppedCount uint32) *octrace.Span_Link
 }
 
 func statusToOC(status data.SpanStatus) *octrace.Status {
-	if status == emptyStatus {
```

Comment:
This did not work properly, because this was doing a pointer comparison, and even if the point to empty structs, they point to different addresses in memory where these empty structs were saved.